src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -58,52 +56,48 @@
 import org.reactome.addlinks.referencecreators.NCBIGeneBasedReferenceCreator;
 import org.reactome.addlinks.referencecreators.OneToOneReferenceCreator;
 import org.reactome.addlinks.referencecreators.RHEAReferenceCreator;
+import org.reactome.addlinks.referencecreators.TargetPathogenReferenceCreator;
 import org.reactome.addlinks.referencecreators.UPMappedIdentifiersReferenceCreator;
-import org.reactome.release.common.database.InstanceEditUtils;
 
 
 public class AddLinks
 {
+	private static final String PROTEIN = "PROTEIN";
+	private static final String CTD_PROCESSOR = "CTDProcessor";
+	private static final String BRENDA_UC = "BRENDA";
```

Comment:
Is BRENDA still needed?

Reply:
There is a separate issue for removing all BRENDA-related code: https://github.com/reactome/add-links/issues/125